packages/wrangler/src/logger.ts

```diff
@@ -51,7 +40,7 @@ function getLoggerLevel(): LoggerLevel {
 		const expected = Object.keys(LOGGER_LEVELS)
 			.map((level) => `"${level}"`)
 			.join(" | ");
-		warnOnce(
+		logger.warnOnce(
```

Comment:
I'm not sure we should be using `logger` methods inside `getLoggerLevel()` like this—it feels like a bit of a circular dependency

Reply:
I thought that may have been the reason. But this function is about to override the default log level – since the default log level will not swallow warnings, I think it's okay to use logger.warn

Reply:
Is it not going to be slightly circular though? `logger.warnOnce()` will eventually call `getLoggerLevel()` which will then call `logger.warnOnce()` again and so on. I might be misunderstanding, but that would be my expectation

Reply:
Yeah I guess. But luckily logger.warnOnce only calls logger.warn... once 😉